src/foam/nanos/auth/User.js

```diff
@@ -77,24 +77,28 @@ foam.CLASS({
       name: 'id',
       documentation: 'The ID for the User.',
       final: true,
-      tableWidth: 50
+      tableWidth: 50,
+      permissionRequired: true
```

Comment:
what does setting these permissions do the Ablii UI? Does everything just work after you make these changes?